jlm/util/Statistics.cpp

```diff
@@ -1,15 +1,230 @@
 /*
  * Copyright 2020 Nico Reißmann <nico.reissmann@gmail.com>
+ * Copyright 2024 Håvard Krogstie <krogstie.havard@gmail.com>
  * See COPYING for terms of redistribution.
  */
 
 #include <jlm/util/Statistics.hpp>
 
+#include <jlm/util/strfmt.hpp>
+
 namespace jlm::util
 {
 
 Statistics::~Statistics() = default;
 
+std::string
+Statistics::SerializeMeasurements() const
```

Comment:
This method does more than the name suggests (it also serializes the timers). How about just naming it `Serialize()`?